src/Microsoft.Restier.Core/ApiBuilder.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.Framework.DependencyInjection;
```

Comment:
And the other thing is EF team will change the package name in near future, so anyway we have to make a change to adapt later. So let's skip it this time.